logic/src/commonMain/kotlin/com/wire/kalium/logic/util/TimeParser.kt

```diff
@@ -4,6 +4,7 @@ import kotlinx.datetime.DateTimeUnit
 import kotlinx.datetime.toInstant
 import kotlinx.datetime.until
 
+//TODO: we need to think if this TimeParser should return an either, should we catch the exception ?
 interface TimeParser {
     fun calculateMillisDifference(time1: String, time2: String): Long
```

Comment:
I would add the comment to make sure everyone using it knows that the datetime we use needs to be the ISO-8601 format (e.g. `YYYY-MM-DDTHH:mm:ss.SSSZ`)

Reply:
will do :)